src/components/MainPanel/WaveformStrip.scss

```
@@ -0,0 +1,36 @@
+// Flat tinted background matching the project's existing visualization
+// panels (.visualization-canvas style from the pre-refactor advanced footer).
+// No gradient, no in-canvas label — color tint alone identifies the channel.
+.waveform-strip {
+  height: 28px;
+  border-radius: 4px;
+  position: relative;
+  overflow: hidden;
+  flex-shrink: 0;
+
+  &--full { width: 110px; max-width: 140px; }
+  // Half-width strips live inside .waveform-input-group (fixed 110px wide).
```

Stylelint (17.11.1)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)